src/globe/ElevationCoverage.js

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright 2015-2018 WorldWind Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * @exports ElevationCoverage
+ */
+define(['../error/ArgumentError',
+        '../util/Logger',
+        '../geom/Sector'],
+    function (ArgumentError,
+              Logger,
+              Sector) {
+        "use strict";
+
+        /**
+         * Constructs an ElevationCoverage
+         * @alias ElevationCoverage
+         * @constructor
+         * @classdesc When used directly and not through a subclass, this class represents an elevation coverage
+         * whose elevations are zero at all locations.
+         * @param {Number} resolution The resolution of the coverage in meters.
+         * @throws {ArgumentError} If the resolution argument is null, undefined, or zero.
+         */
+        var ElevationCoverage = function (resolution) {
+            if (!resolution) {
+                throw new ArgumentError(
+                    Logger.logMessage(Logger.LEVEL_SEVERE, "ElevationCoverage", "constructor",
+                        "missingResolution"));
+            }
+
+            /**
+             * Indicates the last time this coverage changed, in milliseconds since midnight Jan 1, 1970.
+             * @type {Number}
+             * @readonly
+             * @default Date.now() at construction
+             */
+            this.timestamp = Date.now();
+
+            /**
+             * Indicates this coverage's display name.
+             * @type {String}
+             * @default "Elevations"
+             */
+            this.displayName = "Zero Elevations";
+
+            /**
+             * Indicates whether or not to use this coverage.
+             * @type {Boolean}
+             * @default true
+             */
+            this.enabled = true;
```

Comment:
Changing a coverage's enabled property does not cause the terrain to regenerate. One possible adjustment is to update the coverage timestamp when the enabled property changes:
      
```javascript
Object.defineProperties(ElevationCoverage.prototype, {
    /**
     * Indicates whether or not to use this coverage.
     * @type {Boolean}
     * @default true
     */
    enabled: {
        get: function () {
            return this._enabled;
        },
        set: function (value) {
            this._enabled = value;
            this.timestamp = Date.now();
        }
    }
});
```

Reply:
@pdavidc Hi Dave. I think all your concerns have been addressed. Notes:

1. The areaElevationsForGrid resultFilled computation was also causing a subtle bug, so I changed it to operate the same way as pointElevationsForGrid as you suggested.
2. I changed all the external facing APIs to use degrees for resolution as we discussed. LevelSets still use radians internally.
3. Coverage resolutions and levels have been updated as we discussed.
4. I removed "Earth" from the coverage names. 
5. I had left removeCoverageAtIndex in place as a convenience method. It has now been removed.
6. I changed elevationsForGrid to accept a plain Array. I had mistakenly thought Float64Array was required based on usage in the code and a bad assumption.
7. I added the property code for ElevationCoverage.enabled as you suggested. At first it didn't work, due to ElevationModel.timestamp respecting the enabled flag, but that was probably a mistake anyway. Terrain now seems to regenerate fine. 
8. I addressed the various terminology and doc tweaks you pointed out.